File: src/nl/rubensten/texifyidea/LatexLexerAdapter.java
package nl.rubensten.texifyidea;

import com.intellij.lexer.FlexAdapter;
import nl.rubensten.texifyidea.grammar.LatexLexer;

/**
 * @author Sten Wessel
 */
public class LatexLexerAdapter extends FlexAdapter {

    public LatexLexerAdapter() {
        super(new LatexLexer(null));
    }
}

File: src/nl/hannahsten/texifyidea/LatexLexerAdapter.kt
package nl.hannahsten.texifyidea

import com.intellij.lexer.FlexAdapter
import nl.hannahsten.texifyidea.grammar.LatexLexer

/**
 * @author Sten Wessel
 */
class LatexLexerAdapter : FlexAdapter(LatexLexer(null)) 